api.js

```diff
@@ -25,6 +25,9 @@ function detailsToVisit(details) {
 }
 
 exports.init = function (sbot, config) {
+  if (!config) config = {}
+  if (!config.metafeeds) config.metafeeds = {}
```

Comment:
:fire: / `style`
I would say it's a very bad idea to have plugins mutating the `config` - it's probably a bigger issue with SecretStack that there's no scoped config. hmmm

Reply:
OH - we could just deepFreeze the config Object after it's passed in.
I would feel real happy about that.